.github/workflows/sitemap.yml

```diff
@@ -0,0 +1,60 @@
+on: workflow_call
+
+# * installs sitemap generator
+# * runs generator with priority map and lastmod
+# * saves file output to ui/public/sitemap.xml
+# * commits output to branch
+# * opens pull request
+
+defaults:
+  run:
+    working-directory: ui/public
+
+jobs:
+  sitemap:
+    runs-on: ubuntu-latest
+    name: Generate xml sitemap
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+      - name: Install sitemap-generator-cli
+        run: npm install -g sitemap-generator-cli
+      - name: Generate sitemap
+        id: sitemap
+        run: |
+          npx sitemap-generator-cli -lqgv --verbose -p "1.0,0.8,0.6,0.4,0.2" https://data.standards.nhs.uk
```

Comment:
Might be worth a comment around here to explain some of the flags/options. At the very least a link to the docs.

Reply:
Good shout, will link 👍